runtime/src/disable.rs

```diff
@@ -0,0 +1,44 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{Drivers, MailboxResp};
+use caliptra_common::keyids::{KEY_ID_RT_CDI, KEY_ID_RT_PRIV_KEY};
+use caliptra_drivers::{
+    hmac384_kdf, CaliptraError, CaliptraResult, KeyReadArgs, KeyUsage, KeyWriteArgs,
+};
+
+pub struct DisableAttestationCmd;
+impl DisableAttestationCmd {
+    pub(crate) fn execute(drivers: &mut Drivers) -> CaliptraResult<MailboxResp> {
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_CDI)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_PRIV_KEY)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+
+        // dice key is derived from an empty CDI slot so it will not match the key that was certified in the rt_alias cert
```

Comment:
nit: I would recommend putting thing comment as a doc comment on the generate_dice_key definition.